src/scripting/lua_menu_item.cpp

```diff
@@ -26,17 +27,170 @@ static lg::log_domain log_scripting_lua("scripting/lua");
 #define WRN_LUA LOG_STREAM(warn, log_scripting_lua)
 #define ERR_LUA LOG_STREAM(err, log_scripting_lua)
 
-static const char menuitemKey[] = "menu_item";
+/**
+ * Implementation for a lua reference to a unit_type.
```

Comment:
Looks like this comment needs updating.

Reply:
Ah, yes.   There's also some missing comments starting at line 154, I wasn't sure that to put there.  And the set for _dir, which I copied from lua_unit.cpp, don't look right to me.